pyproject.toml

```diff
@@ -95,7 +96,7 @@ serial = [
 ]
 
 http2 = [
-    "h2 >= 3.0, < 5.0",
+    "h2 >= 3.2, < 5.0",
```

Comment:
Necessary to keep httpx happy in the tests. 3.2.0 is from February 2020, still plenty of leeway.